src/style.css

```diff
@@ -0,0 +1,102 @@
+@import "~@fortawesome/fontawesome-free/css/all.css";
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  background-color: rgba(179, 172, 172, 0.137);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding-top: 50px;
+  padding-left: 25px;
+
+  /* optional: center vertically as well */
+  width: 600px;
+  margin: 100px auto;
+  border-right: 50px solid rgba(179, 172, 172, 0.137);
+  border-left: 50px solid rgba(179, 172, 172, 0.137);
+  border-top: 10px solid rgba(179, 172, 172, 0.137);
+  border-bottom: 10px solid rgba(179, 172, 172, 0.137);
+  border-radius: 5px;
+}
+
+/* .far {
+  padding-top: 15px;
+  background-color: white;
+} */
```

Comment:
- [ ] Let's try not to have uncommented code in our codebase as this makes our code dirty and unprofessional. It's a good convention to not have commented unused code in our codebase. Please remove them 👍🏽